common/lib/opentelemetry/common/utilities.rb

```diff
@@ -46,6 +46,10 @@ def utf8_encode(string, binary: false, placeholder: STRING_PLACEHOLDER)
         placeholder
       end
 
+      def truncate(string, size)
```

Comment:
This method exists in a few places now, will move the other instances to this one in a separate smaller PR.